test/cmdlineTests/asm_json_import_all_valid_flags/output

```diff
@@ -22,8 +22,8 @@
                         ".code":
                         [
                             {
-                                "begin": 0,
-                                "end": 0,
+                                "begin": -1,
```

Comment:
Those changes are just here because of a bug during evm assembly import. If no source locations where defined in the input it was setting it to `0` instead of leaving it undefined `-1`

Reply:
See [related change](https://github.com/ethereum/solidity/pull/14877/files#diff-786911c6e3eab4698a9fab73ced71946ad895b439784b26b05a0321c2ae0e9ceR127).

Reply:
I honestly would have preferred this in another PR (this one is already large enough as is) - also, is the evm assembly import experimental? If not, this would warrant a bugfix changelog entry, right?

Reply:
I guess you're right!

Reply:
Created https://github.com/ethereum/solidity/pull/14903 and rebased this one on top of that.